lib/mongo/socket.rb

```diff
@@ -186,6 +203,11 @@ def read_from_socket(length)
       data
     end
 
+    def read_buffer_size
+      # Buffer size for non-SSL reads
+      65536
```

Comment:
To help any readers who haven't memorized the powers of two, I think a comment here explaining that this is 64kb (mirroring the comment in the SSL socket file) would be useful.